lamindb/models/artifact.py

```diff
@@ -1103,6 +1104,9 @@ class Meta(SQLRecord.Meta, IsVersioned.Meta, TracksRun.Meta, TracksUpdates.Meta)
             ),
         ]
 
+    _aux_fields: dict[str, tuple[str, type]] = {
```

Comment:
We need to set the default value to `False` in the Artifact `__init__`.

Reply:
I think it needs to be called after super().__init__(), parallel to here

https://github.com/laminlabs/lamindb/blob/cf5ede63b2abc15e8960830b0afd0a6f81d31636/lamindb/models/feature.py#L636-L638

Reply:
See the answer to the first comment.

Reply:
Ok.